### ISSUE TITLE
Docker build suddenly fails

### ISSUE THREAD

**Issue by justb4** [geopython/GeoHealthCheck#240]
Without any change to code or setup config, suddenly build failure (and demo site will not come up):

Has to do with `proj.4` build error. Very strange.

```
Requirement already satisfied: pytz in /venv/lib/python2.7/site-packages (from OWSLib==0.17.0->-r requirements.txt (line 8)) (2018.9)
Collecting pyproj (from OWSLib==0.17.0->-r requirements.txt (line 8))
  Downloading https://files.pythonhosted.org/packages/53/4b/21643a93e7d33941498087290636e34c6d534aa8baa1ada54cf0d096ffaa/pyproj-2.0.0.tar.gz (408kB)
  Installing build dependencies: started
  Installing build dependencies: still running...
  Installing build dependencies: finished with status 'done'
  Getting requirements to build wheel: started
  Getting requirements to build wheel: finished with status 'error'
  Complete output from command /venv/bin/python /venv/lib/python2.7/site-packages/pip/_vendor/pep517/_in_process.py get_requires_for_build_wheel /tmp/tmpMbg3IO:
  Proj executable not found. Please set PROJ_DIR variable.
  
  ----------------------------------------
Command "/venv/bin/python /venv/lib/python2.7/site-packages/pip/_vendor/pep517/_in_process.py get_requires_for_build_wheel /tmp/tmpMbg3IO" failed with error code 1 in /tmp/pip-install-eo2MIh/pyproj


Captured Task Output:
---------------------

---> pavement.setup
pip install -r requirements.txt

Build failed running pavement.setup: Subprocess return code: 1

```

**Comment by tomkralidis** [comment]
This is a proj error buried down in https://github.com/jswhit/pyproj/issues/177 against pyproj 2.0.0 which was released yesterday.

**Comment by justb4** [comment]
Ok, clear, I see https://github.com/geopython/OWSLib/blob/master/requirements.txt has no version pinned for pyproj. Not sure if that was intended but it can make builds like above non-reproducible.

**Comment by justb4** [comment]
I will fix for now by setting `pyproj==1.9.6` in GHC requirements.txt, so we have at least the build and demo site working.

**Comment by justb4** [comment]
Seems Dependance Day today :-(, latest Docker build hit by silent upgrade of SQLAchemy from 1.2.* to 1.3.0 (released 2019-03-04) . Error only happens when clicking in left menu on a Resource Type.

![image](https://user-images.githubusercontent.com/582630/54072298-5bcabc80-4279-11e9-9323-edac8b3111ef.png)
 
Will work further on the requirements.txt...Using SQlAlchemy 1.2.18 (latest 1.2 version) we see these errors as warnings: *SAWarning: Textual SQL expression u"resource_type = 'OSGeo:TM..." should be explicitly declared as text(u"resource_type = 'OSGeo:TM...") (this warning may be suppressed after 10 occurrences)* This harks back to [this line in views.py](https://github.com/geopython/GeoHealthCheck/blob/master/GeoHealthCheck/views.py#L64). Need to dive into for fix but for now will pin SA to 1.2.18 to get demo site ok.

**Comment by justb4** [comment]
Build ok and https://demo.geohealthcheck.org working again. Open separate issue to pin versions more strictly.
